doc/classes/TabBar.xml

```diff
@@ -140,6 +140,20 @@
 				Removes the tab at index [param tab_idx].
 			</description>
 		</method>
+		<method name="select_next_available">
+			<return type="bool" />
+			<description>
+				Selects first available tab on the left from the currently selected. Returns [code]true[/code] if tab selection changed.
+				Uses value of the [member tab_input_rollover] property to determine if selection should rollover to the tab on the most right if there's no more available tabs on the left.
+			</description>
+		</method>
+		<method name="select_previous_available">
+			<return type="bool" />
+			<description>
+				Selects first available tab on the right from the currently selected. Returns [code]true[/code] if tab selection changed.
+				Uses value of the [member tab_input_rollover] property to determine if selection should rollover to the tab on the most left if there's no more available tabs on the right.
+			</description>
+		</method>
```

Comment:
Descriptions are switched around. "On the left" should be for the previous available, and "on the right" for the next available. Implementations seem to be correct in that regard, so it's just the docs that needs fixing, in both classes.

Reply:
Oops, this is what I get for using diff tool and not noticing the methods switched alphabetical order.